### PR TITLE
V1.0.32

### PR DESCRIPTION
**Pull request by rmomii**
@brozeph Adding this functionality to support clean-orphaned-locations

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1494,6 +1495,35 @@ client
...
+  * `err` - populated with details in the event of an error
+  * `physicalLocation` - the physicalLocation
+
+```javascript
+var physicalLocationId = '<PHYSICAL_LOCATION_ID>';
+
+client
+  .location
+  .deletePhysicalLocation(physicalLocationId)
+  .then((physicalLocation) => {
```
Comment (brozeph) [inline readme.md]: We may want to adjust this line here - my guess is that the physical location that was removed is not actually returned (i.e. the response is a `204` with no content)

Comment (brozeph) [reply readme.md]: i.e.

``` javascript
client
  .location
  .deletePhysicalLocation(physicalLocationId)
  .then(() => {
  // .. etc.
```

**Review by brozeph — CHANGES_REQUESTED** [review]
Looks excellent so far... if you can make a quick adjustment on the documentation, I'll merge and publish the update to NPM!

File: readme.md (hunk 1 above)
Comment (rmomii) [reply readme.md]: Got it, just committed it.

**Review by rmomii — COMMENTED** [review]
Looks good

**Comment by rmomii** [comment]
@brozeph lol I can't figure out Githubs new review thing but I just committed the change to the docs.

**Review by brozeph — APPROVED** [review]

**Comment by brozeph** [comment]
Now published to NPM as `v1.0.32`

**Pull request merged**